integrations/pytorch/tutorials/woodfisher_pruning.md

```diff
@@ -0,0 +1,126 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# WoodFisher Pruning with SparseML
+
+This tutorial shows how to use the WoodFisher pruning algorithm in a SparseML PyTorch
+pruning recipe. The WoodFisher algorithm was introduced in the paper 
+[WoodFisher: Efficient Second-Order Approximation for Neural Network Compression](https://arxiv.org/pdf/2004.14340.pdf)
+by Sidak Pal Singh and Dan Alistarh.
+
+## Overview
+The WoodFisher algorithm introduces an efficient process for approximating accurate
+second-order information about a model's weights that can be used to solve for optimal
+weights to prune.  The method uses the weights' first-order information
+(gradients) to approximate the Inverse-Hessian matrix that is used to score weights
+for pruning.
+
+This method was shown to achieve SoTA results for pruning tasks on
+image classification models such as ResNet and MobileNet. Detailed information of the
+methods and results can be found in the [paper](https://arxiv.org/pdf/2004.14340.pdf).
+
+WoodFisher pruning can be enabled by a single modifier in a SparseML recipe. This tutorial
+will demonstrate how to adapt an existing SparseML pruning recipe to do this.
+
```

Comment:
```suggestion

## Need Help?
For Neural Magic Support, sign up or log in to get help with your questions in our **Tutorials channel:** [Discourse Forum](https://discuss.neuralmagic.com/) and/or [Slack](https://join.slack.com/t/discuss-neuralmagic/shared_invite/zt-q1a1cnvo-YBoICSIw3L1dmQpjBeDurQ). 

```

Reply:
added to bottom

Reply:
For the template, we're intentionally upfront letting people know that if they need any help before they actually start the tutorial, there's a method to do it. We then remind them at the end as a wrap-up, to not forget about the offer to help.

Reply:
moved it up